src/main/java/dev/meyi/bn/modules/calc/CraftingCalculator.java

```
@@ -263,10 +258,11 @@ public static String[] getEnchantCraft(String itemU) {
   }
 
   public static void getUnlockedRecipes() {
-    try {
-      unlockedRecipes = Utils.unlockedRecipes().toString();
-    } catch (IOException ignored) {
-    }
+      try {
+        String s = Utils.unlockedRecipes().toString();
+        if (s != null) {
+          unlockedRecipes = s;
+        }
+      } catch (IOException ignored) {}

```

I know I'm saying this a lot, but let's not just hide these exceptions, `ignored.printStackTrace()`